docs/resources/environments_azure_environment.md

```diff
@@ -53,14 +53,23 @@ resource "cdp_environments_azure_environment" "example-env" {
   resource_group_name              = "rg-name"
   encryption_user_managed_identity = "some-identity"
   use_public_ip                    = true
+  compute_cluster = {
+    enabled = false
+    configuration = {
+      private_cluster = false
+      outbound_type   = "udr"
```

Comment:
What is the point of having this as an attribute with a single possible value?

Reply:
on the public API, this is an enum which, as of now, has a single possible value, but it doesn't mean that it cannot be expanded later on (otherwise it wouldn't be a flag there in the first place). so I kept it as a string and if the user would give something else, then the request would be rejected from the public API with a meaningful reason message